providers/mdxProvider.tsx

```diff
@@ -1,14 +1,20 @@
 import { MDXProvider as BaseMDXProvider } from '@mdx-js/react';
 import type { MDXComponents } from 'mdx/types';
 import { MDXRemote } from 'next-mdx-remote';
+import type { ComponentProps, FC } from 'react';
 import { useEffect } from 'react';
-import type { FC } from 'react';
 
+import { MDXCodeBox } from '@/components/Common/CodeBox';
+import { MDXCodeTabs } from '@/components/Common/CodeBox/CodeTabs';
 import NodeApiVersionLinks from '@/components/Docs/NodeApiVersionLinks';
 
-const mdxComponents: MDXComponents = {
+export const mdxComponents: MDXComponents = {
   NodeApiVersionLinks: NodeApiVersionLinks,
   blockquote: ({ children }) => <div className="highlight-box">{children}</div>,
+  pre: ({ children }) => <MDXCodeBox>{children}</MDXCodeBox>,
+  codeTabs: ({ children, ...props }: ComponentProps<typeof MDXCodeTabs>) => (
```

Comment:
Code Tabs is not a native Markdown element. What is this and why is it needed here?

If the point is to make it easy to invoke `<CodeTabs />` from any MDX document, then it should be `CodeTabs` not `codeTabs`